src/internal/migrations/migrations.go

```diff
@@ -18,14 +20,18 @@ type Env struct {
 	// TODO: etcd
 	ObjectClient obj.Client
 	Tx           *pachsql.Tx
+	EtcdClient   *clientv3.Client
+	EtcdPrefix   string
```

Comment:
I don't think this makes sense in the migrations.Env object.  There is one etcd per cluster, we should be using the whole thing right?